api/v1alpha1/fenceagentsremediation_webhook_test.go

```diff
@@ -11,58 +11,118 @@ var _ = Describe("FenceAgentsRemediation Validation", func() {
 
 	Context("creating FenceAgentsRemediation", func() {
 
-		When("agent name match format and binary", func() {
-			It("should be accepted", func() {
-				far := getTestFAR(validAgentName)
-				_, err := far.ValidateCreate()
-				Expect(err).ToNot(HaveOccurred())
+		Context("with ResourceDeletion strategy", func() {
```

Comment:
misleading description, agent name is always validated. I think this context can be deleted, and its content just moves a level up

Reply:
Fixed it.